app/views/miq_request/_request.html.haml

```diff
@@ -14,23 +14,23 @@
       %label.control-label.col-md-2
         = _("Status")
       .col-md-8
-        = h(@miq_request.status)
+        = h(_(@miq_request.status))
     .form-group
       %label.control-label.col-md-2
         = _("Request State")
       .col-md-8
-        = h(@miq_request.state.titleize)
+        = h(_(@miq_request.state.titleize))
     - if @miq_request.requester
       .form-group
         %label.control-label.col-md-2
           = _("Requester")
         .col-md-8
-          = h(@miq_request.requester_name)
+          = h(_(@miq_request.requester_name))
```

Comment:
We cannot translate `@miq_request.requester_name`, can we? What does it contain?